### DISCUSSION TITLE
[BUG] Review Status Mixed Up

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1906] [Support]
So, looking at a review page for a Pub, it's saying the opposite of the actual status for each review (or just taking me to the wrong review on click):

![image](https://user-images.githubusercontent.com/3883763/160288436-4d0c9537-22d4-41ca-a5e8-336689a6ee00.png)

When I click R4, it takes me to R1: 

![image](https://user-images.githubusercontent.com/3883763/160288480-49c96da7-41e7-47a0-93b5-5ecf3ef82852.png)

Ideas?

**Comment by idreyn** [comment]
Uh-oh. We'll take a look at this promptly.

**Comment by writingpoli** [comment]
Looks like this may be fixed.
